src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -52,6 +53,7 @@
 
 public class ReceivedMessage extends AbstractMessage
 {
+    public static boolean contentIntentWarning = false;
```

Comment:
can we make this a more boolean-y name?
`shouldShowContentIntentWarning` (which ofc would flip the actual meaning, but still)